presto-main/src/main/java/com/facebook/presto/sql/rewrite/ShowQueriesRewrite.java

```diff
@@ -595,16 +596,6 @@ protected Node visitShowCreateFunction(ShowCreateFunction node, Void context)
                     ordering(ascending("argument_types")));
         }
 
-        private QualifiedName getQualifiedName(ShowCreate node, QualifiedObjectName objectName)
```

Comment:
@v-jizhang looks like this is getting the parts for the name backwards.

Reply:
Trino still has the same code
https://github.com/trinodb/trino/blob/99ef4ec162b3b099762e4d6727ee3db641e1445c/core/trino-main/src/main/java/io/trino/sql/rewrite/ShowQueriesRewrite.java#L593

This PR was requested by some users, I'll do a fix and resubmit.